project-templates/java-emf-theia/glsp-server/src/main/java/org/eclipse/glsp/example/javaemftheia/emf/EMFResourceManager.java

```diff
@@ -0,0 +1,49 @@
+/********************************************************************************
+ * Copyright (c) 2022 EclipseSource and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+package org.eclipse.glsp.example.javaemftheia.emf;
+
+import org.eclipse.emf.common.command.Command;
+import org.eclipse.emf.ecore.EObject;
+import org.eclipse.emfcloud.modelserver.glsp.notation.Diagram;
+
+/**
+ * This resource manager allows to load, save and manipulate EMF resources.
+ */
+public interface EMFResourceManager {
```

Comment:
As mentioned above, I'm not quite sure we should introduce another concept here.
This class combines
1. the SourceModelStorage functionality which could for EMF operate on ResourceSet level
2. the CommandStack, which could just be specialized (if necessary) and used by the gmodel state and all others as they do by default

Reply:
Yes I agree, the resource manager was removed with the follow-up commit.